docs/rfc/README.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 0
```

Comment:
My bad @tac0turtle , it was sidebar_position: 1 needed here.
![image](https://user-images.githubusercontent.com/29894366/224439418-603f5865-4a96-4061-97a2-a67f72ba7e06.png)
As it is a main section, setting 0 makes it as _important_ as the readme :/